### DISCUSSION TITLE
Fail2Ban v1.0.1: ERROR  No 'host' group in '<cmnfailre-failed-pub-<publickey>>

### DISCUSSION THREAD

**Discussion by flat-linex** [fail2ban/fail2ban#3369] [Q&A] [answered]
I'm trying to use version Fail2Ban v1.0.1 on an old Ubuntu 14.04 replacing the original old version. I removed the old /etc/fail2ban contents, installed the new version as explained in the entry page and copied the debian file to replace the original /etc/init.d/fail2ban.

However, when I start the service I get a fail and the error log reports:
`ERROR  No 'host' group in '<cmnfailre-failed-pub-<publickey>>`

What am I missing?
Thanks a lot,

**Comment by sebres** [comment, marked as answer]
> What am I missing?

I'm guess it is still a version conflict - the init script starts some old version, e. g. from `/usr/bin` vs `/bin` or vise versa.
Which version do you see in `fail2ban.log` if it starts? What do you see by `fail2ban-client --version`.

Basically you had to remove previous version before you install something from source.
Now you need to check where is your new `fail2ban-client` you installed from source.
Try to check it with `whereis fail2ban-client` and if found twice, find out the correct one with `fail2ban-client --version` (and delete old version for pattern `/some-path/fail2ban-*`).

Also note that running as init-script a daemon may have another PATH variable as if it gets started from interactive shell.

Another thing would be double python module (somewhere in output of `python -c 'import sys; print("\n".join(sys.path))'` you may find two different fail2ban-modules). For instance try something like this:
```
find `python -c 'import sys; print(" ".join(sys.path))'` -name 'fail2ban' -type d
```
(you may change `python` to `python3` or `python2`, depending on which is default on your system and which you used to install it)

**Reply by flat-linex** [reply]
That was it! I completely removed previous Ubuntu install and re-installed the new version which is now running.
Thanks a lot.
